ivc/src/ivc/columns.rs

```diff
@@ -25,6 +25,9 @@ pub fn block_height<const N_COL_TOTAL: usize, const N_CHALS: usize>(block_num: u
 pub const IVC_POSEIDON_STATE_SIZE: usize = 3;
 pub const IVC_POSEIDON_NB_FULL_ROUND: usize = 55;
 
+pub const IVC_NB_TOTAL_FIXED_SELECTORS: usize =
+    IVC_POSEIDON_NB_FULL_ROUND * IVC_POSEIDON_STATE_SIZE + N_BLOCKS;
```

Comment:
Cosmetic: maybe create  `N_FSEL_POSEIDON = IVC_POSEIDON_NB_FULL_ROUND * IVC_POSEIDON_STATE_SIZE` and refer to that?

Reply:
Generally where possible it's good to keep to the same notation `prover` is using: `N_WIT, N_FSEL, N_DSEL, N_REL` etc